Resources/Computing/ARCComputingElement.py

```diff
@@ -180,6 +181,9 @@ def submitJob( self, executableFile, proxy, numberOfJobs = 1 ):
     """ Method to submit job
     """
 
+    # Assume that the ARC queues are always of the format nordugrid-<batchSystem>-<queue>
+    # And none of our supported batch systems have a "-" in their name
+    self.arcQueue = self.queue.split("-",2)[2]
```

Comment:
Going over all the CEs in the configuration, this is 100% true. I have also asked the nordugrid guys, but they may take time to reply.